systers_portal/templates/403.html

```diff
@@ -9,9 +9,16 @@
     <h1>Access Denied</h1>
     <p>Unfortunately, you don't have permission to access this page.</p>
     <p>
-    	<a class="btn btn-lg btn-success" href="#" role="button" style="margin-top: 80px" onclick="history.go(-1); return false;">Back to previous page</a>
+        <a id="back-button" class="btn btn-lg btn-success"role="button" style="margin-top: 80px" onclick="history.go(-1); return false;">Back to previous page</a>
```

Comment:
This id is better. We don't need ```onclick``` as I shared before.

Reply:
I used onclick = history.go(-1) to go back to the previous page without accessing history (security problem) - but requesting the browser to take us to the previous page.Without onclick, the button wouldn't work.
For hover to work,  the href='<value>' has to be set and it's impossible to get the previous page link from the onclick so I cannot remove any one of them. :( 
I have done rebasing, and I am struggling to rebase further, since the changes are through different paths. Could you help?

Reply:
In which browser are you testing?

Reply:
I am testing on firefox.

Reply:
Thanks for making it work for all the cases that I was missing out.